src/Azure/AzureAD/Authentication.AzureADB2C.UI/src/Microsoft.AspNetCore.Authentication.AzureADB2C.UI.csproj

```diff
@@ -6,7 +6,6 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <PackageTags>aspnetcore;authentication;AzureADB2C</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <ProvideApplicationPartFactoryAttributeTypeName>Microsoft.AspNetCore.Mvc.ApplicationParts.NullApplicationPartFactory, Microsoft.AspNetCore.Mvc.Core</ProvideApplicationPartFactoryAttributeTypeName>
```

Comment:
Is this a new feature?

Reply:
`NullApplicationPartFactory`? No, this was a workaround that was in place to support the old Azure AD UI scenarios.

A user could call an extension method `AddAzureAd` to configure the app areas and controller routes that would set up signin.

Razor ships with a `CompiledRazorItemApplicationPartFactory` that discovers and loads view types in the related assembly.

To support the extension method scenario, this project was circumventing the `CompiledRazorItemApplicationPartFactory` logic and doing the loading on its own in the extension method.

We've removed the concept of a related assembly with the consolidated views work so this is no longer needed.